tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.cpp

```diff
@@ -167,22 +167,10 @@ extern "C" DLL_EXPORT BOOL WINAPI MarshalStructAsParam_AsSeqByValOut3(CharSetAns
 }
 extern "C" DLL_EXPORT BOOL WINAPI MarshalStructAsParam_AsSeqByRefOut3(CharSetAnsiSequential* str1)
 {
-	char const* strSource = "change string";
```

Comment:
This kind of code is everywhere and all it does is copying a copy of a string. This belongs in a separate function.